### PR TITLE
Feature 2.0 bump

### PR DESCRIPTION
**Pull request by LobeTia**
Bumped to trails 2.0

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,19 +23,19 @@
...
-    "mocha": "^2.4",
-    "smokesignals": "^1.2.2",
-    "trailpack-core": "^1.0.0-beta-6",
-    "trails": "^1.0.0",
-    "trails-model": "^1.0.0-beta-2"
+    "eslint": "3.13.1",
+    "eslint-config-trails": "2.0.6",
+    "mocha": "3.2.0",
+    "smokesignals": "2.1.0",
+    "trailpack-core": "2.0.0",
```
Comment (jaumard) [inline package.json]: this one is not needed anymore for v2 :) you can remove it

File: package.json [hunk 2]
```diff
@@ -23,19 +23,19 @@
...
+    "trails": "2.0.0",
+    "trails-model": "2.0.1"
```
Comment (jaumard) [inline package.json]: Classes are not needed anymore to, you can remove them and use `require('trails/model')`

Comment (LobeTia) [reply package.json]: Totally missed this feature, great change!

**Comment by konstantinzolotarev** [comment]
@LobeTia Awesome. Thank you very much !

**Pull request merged**